pagopa/pn-frontend-e2e
# Feat/view annullamento pg

## Short description

Test to check the download in a cancelled notification

## List of changes proposed in this pull request

- created a unique test to check the correct cancelled notification displays for PG
- reused the method created for PF
- implemented some new steps to match the tasks requirements

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@checkDownloadFileNotificaAnnullataPG"` and verify the test works fine.
